tests/validation/tests/v3_api/test_rke_cluster_provisioning.py

```diff
@@ -1,31 +1,11 @@
-import os
 from threading import Thread
 import pytest
 from .common import *  # NOQA
 from rancher import ApiError
 
-K8S_VERSION = os.environ.get('RANCHER_K8S_VERSION', "")
-K8S_VERSION_UPGRADE = os.environ.get('RANCHER_K8S_VERSION_UPGRADE', "")
-POD_SECURITY_POLICY_TEMPLATE = os.environ.get('RANCHER_POD_SECURITY_POLICY_TEMPLATE', "restricted")
-DO_ACCESSKEY = os.environ.get('DO_ACCESSKEY', "None")
-LINODE_ACCESSKEY = os.environ.get('RANCHER_LINODE_ACCESSKEY', "None")
-AWS_ACCESS_KEY_ID = os.environ.get("AWS_ACCESS_KEY_ID")
-AWS_SECRET_ACCESS_KEY = os.environ.get("AWS_SECRET_ACCESS_KEY")
-AWS_REGION = os.environ.get("AWS_REGION")
-AWS_SUBNET = os.environ.get("AWS_SUBNET")
-AWS_VPC = os.environ.get("AWS_VPC")
-AWS_SG = os.environ.get("AWS_SG")
-AWS_ZONE = os.environ.get("AWS_ZONE")
-AWS_IAM_PROFILE = os.environ.get("AWS_IAM_PROFILE", "")
-AZURE_SUBSCRIPTION_ID = os.environ.get("AZURE_SUBSCRIPTION_ID")
-AZURE_CLIENT_ID = os.environ.get("AZURE_CLIENT_ID")
-AZURE_CLIENT_SECRET = os.environ.get("AZURE_CLIENT_SECRET")
-AZURE_TENANT_ID = os.environ.get("AZURE_TENANT_ID")
-worker_count = int(os.environ.get('RANCHER_STRESS_TEST_WORKER_COUNT', 1))
-HOST_NAME = os.environ.get('RANCHER_HOST_NAME', "testcustom")
-
```

Comment:
Can you move out only the AWS and linode related variiables out to common.py and leave the rest of them here.